datastore/src/main/java/com/openlattice/datastore/data/controllers/DataController.java

```diff
@@ -1059,4 +1083,32 @@ private void ensureEntitySetsCanBeWritten( Set<UUID> entitySetIds ) {
         }
     }
 
+    private int blockOnDeletionJobGettingNumUpdates( UUID deletionJobId, boolean blockUntilCompletion ) {
+        if ( !blockUntilCompletion ) {
+            return 0;
+        }
+
+        for ( int i = 0; i < MAX_DELETION_BLOCKING_CHECKS; i++ ) {
```

Comment:
`MAX_DELETION_BLOCKING_CHECKS` is defined as 

```
MAX_DELETION_BLOCKING_CHECKS = DELETION_BLOCKING_INTERVAL * 60 * 5; // 5 minutes
```

... which means the for-loop will run for 300,000 iterations. Since there's a `Thread.sleep` for 1 second, doesn't that mean the for-loop should only do 300 iterations? 

In other words, shouldn't `i++` actually be `i += 1000`?

Reply:
ah yeah good catch!